scripts/generateProptypes.ts

```diff
@@ -324,47 +210,25 @@ async function run(argv: HandlerArgv) {
     console.log(`Only considering declaration files matching ${filePattern}`);
   }
 
-  const buildProject = createTypeScriptProjectBuilder(CORE_TYPESCRIPT_PROJECTS);
+  const buildProject = createTypeScriptProjectBuilder(PIGMENT_CSS_TYPESCRIPT_PROJECTS);
 
   // Matches files where the folder and file both start with uppercase letters
   // Example: AppBar/AppBar.d.ts
   const allFiles = await Promise.all(
-    [
-      path.resolve(__dirname, '../packages/mui-system/src'),
-      path.resolve(__dirname, '../packages/mui-base/src'),
-      path.resolve(__dirname, '../packages/mui-material/src'),
-      path.resolve(__dirname, '../packages/mui-lab/src'),
-      path.resolve(__dirname, '../packages/mui-joy/src'),
-    ].map((folderPath) =>
-      glob('+([A-Z])*/+([A-Z])*.*@(d.ts|ts|tsx)', {
+    [path.resolve(__dirname, '../packages/pigment-css-react/src')].map((folderPath) =>
+      glob('+([A-Z])*.*@(d.ts|ts|tsx)', {
         absolute: true,
         cwd: folderPath,
       }),
     ),
   );
-
-  const files = _.flatten(allFiles)
-    .filter((filePath) => {
-      // Filter out files where the directory name and filename doesn't match
-      // Example: Modal/ModalManager.d.ts
-      let folderName = path.basename(path.dirname(filePath));
-      const fileName = path.basename(filePath).replace(/(\.d\.ts|\.tsx|\.ts)/g, '');
-
-      // An exception is if the folder name starts with Unstable_/unstable_
-      // Example: Unstable_Grid2/Grid2.tsx
-      if (/(u|U)nstable_/g.test(folderName)) {
-        folderName = folderName.slice(9);
-      }
-
-      return fileName === folderName;
-    })
-    .filter((filePath) => filePattern.test(filePath));
+  const files = _.flatten(allFiles).filter((filePath) => filePattern.test(filePath));
```

Comment:
The components in Pigment CSS are defined in the first-level directory, not nested ones, so I changed the script to address this.